pytest-embedded-qemu/pyproject.toml

```
@@ -30,6 +30,7 @@ requires-python = ">=3.7"
 
 dependencies = [
     "pytest-embedded~=1.4.2",
+    "qemu.qmp"
```

```suggestion
    "qemu.qmp==0.0.3"
```

since this package is still under early dev phase (0.0.x version), I recommend to lock the version here.

---

Changed